conf.py

```diff
@@ -143,6 +143,15 @@
 
 html_theme = 'sphinx_rtd_theme'
 
+# Set canonical URL from the Read the Docs Domain
+html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "")
```

Comment:
Do we [actually need to set this](https://docs.readthedocs.io/en/stable/guides/canonical-urls.html#sphinx)? Asking since we do not currently set it for the Spicy docs.

Reply:
Hmm.. the email said:

> Are my projects affected?
> You have at least one project with a canonical custom domain defined. After the deprecation date, Read the ? Docs will stop defining [html_baseurl](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_baseurl) Sphinx configuration, which means that you will need to define it by yourself to keep the canonical custom domain properly configured.
> 
> We found the following projects where you are a maintainer are impacted by this change:
>
> [zeek-docs](https://readthedocs.org/projects/zeek-docs/)

The docs you linked might be dated?

Spicy is a subproject of Zeek, so maybe there pre-notification hasn't detected that, or it's not needed for subprojects (though that would be confusing, too).


![Screenshot from 2024-07-31 12-04-20](https://github.com/user-attachments/assets/d09a721d-a952-41b0-b29a-f9d69eaa6d6e)